docs/how-to/index.md

```diff
@@ -0,0 +1,7 @@
+---
+has_children: true
+title: How To
+nav_order: 250
```

Comment:
This conflicts with the nav order `recipes`. I remember this causing one of the two sections to disappear.

Reply:
Ah I saw that Recipes and Concepts both had nav_order 250 so thought I'd follow since I want it around the same place

Any idea what the nav_order value should be? I'm not understanding the pattern

Reply:
I think it's a bit like `z-index`, any numeric value to specify an arbitrary order.
Unlike `z-index`, though, I found that of two navs with the same order, one doesn't get displayed.

Reply:
Hmm still confused about why there are already two 250 indexes or what indexes in the hundreds mean but this should work now